File: cloud-collect-core/src/test/java/com/windea/demo/cloudcollect/core/TestController.java
package com.windea.demo.cloudcollect.core;

import org.springframework.data.domain.Pageable;
import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/test")
public class TestController {
	@GetMapping("/string")
	void getString(@RequestParam String string) {
		System.out.println(string);
	}

	@GetMapping("/pageable")
	void getPageable(@RequestParam Pageable pageable) {
		System.out.println(pageable);
	}
}

File: cloud-collect-core/src/test/kotlin/com/windea/demo/cloudcollect/core/TestController.kt
package com.windea.demo.cloudcollect.core

import org.springframework.data.domain.*
import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping("/test")
class TestController {
	@GetMapping("/string")
	internal fun getString(@RequestParam string: String) {
		println(string)
	}
	
	@GetMapping("/pageable")
	internal fun getPageable(@RequestParam pageable: Pageable) {
		println(pageable)
	}
}
